facets_overview/components/facets_overview_chart/facets-overview-chart.html

```diff
@@ -23,7 +23,7 @@
 
 <dom-module id='facets-overview-chart'>
   <template>
-    <link rel="stylesheet" href="../../../tf-imports/plottable.css">
+    <style include="plottable-style"></style>
```

Comment:
where is plottable-style defined? through  "@org_tensorflow_tensorboard//tensorboard/components/tf_imports:plottable"?

Reply:
style#include is a Polymer thing where you can specify name of the dom-module:
https://github.com/stephanwlee/tensorboard/blob/8837c95c5249ed7a643fd6bf5d47ba90125e6742/tensorboard/components/tf_imports/plottable.html#L29-L33

1: https://polymer-library.polymer-project.org/1.0/docs/devguide/styling
2: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style